tests/wpt/meta/streams/transferable/deserialize-error.window.js.ini

```diff
@@ -1,7 +1,2 @@
 [deserialize-error.window.html]
   expected: ERROR
```

Comment:
Note that the error for the suite remains, which is due to an unhandled rejection of a promise, and after some debugging I believe that this will go away with https://github.com/servo/servo/issues/36465